api/spark.py

```
@@ -33,8 +33,12 @@ def do_POST(self):
             result = run_spark(user_id, chat_id, conn, cfg)
             self._respond(200, result)
         except Exception as e:
-            msg = "Something went wrong. Please try again in a moment."
-            send_message(chat_id, msg, cfg.telegram_bot_token)
+            import traceback
+            print(f"[spark] UNHANDLED ERROR: {traceback.format_exc()}")
+            try:
+                send_message(chat_id, "Something went wrong. Please try again in a moment.", cfg.telegram_bot_token)
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)